external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -88,6 +89,8 @@ private[kafka010] class KafkaSource(
   private val maxOffsetsPerTrigger =
     sourceOptions.get("maxOffsetsPerTrigger").map(_.toLong)
 
+  private val VERSION = "v1"
```

Comment:
nit: please move this to `object KafkaSource`. It should be `v1\n` if you are using it in `startsWith`. Otherwise, you cannot distinguish `v1` and `v11`.